JavascriptFormatter.js

```diff
@@ -44,7 +44,7 @@ function filterForRemoteControl(originalCommands) {
             var c1 = c.createCopy();
             c1.command = c.command.replace(/AndWait$/, '');
             commands.push(c1);
-            commands.push(new Command("waitForPageToLoad", options['global.timeout'] || "30000"));
+            commands.push(new Command("waitForPageToLoad", "options.timeout || " + (options['global.timeout'] || "30000")));
```

Comment:
Hi :)

Not sure what you mean. `options['global.timeout']` isn't defined anywhere that I can see, so I assumed it was something from your other project, like the "lbParm" from before?

Or do you mean that I shouldn't allow it to be changed at run-time?

Cheers :)